beamer/health/check.py

```diff
@@ -398,8 +470,8 @@ def analyze_transfers(transfers: TransferMap, ctx: Context) -> None:
 Total fills in network: {ctx.stats.fills}
 Total claims in network: {ctx.stats.claims}
 Total withdrawals in network: {ctx.stats.withdrawals}
-Total total volume in network: {ctx.stats.volume * 1e-6} USDC
-Total expired requests volume in network: {ctx.stats.expired_requests_volume * 1e-6} USDC
+Total total volume in network: {volume_per_token}
```

Comment:
Why this is not just Total volume but Total total volume :D

Reply:
probably a typo, will fix it :D